src/chains/ethereum_1/helpers/generalModelHelpers.ts

```diff
@@ -0,0 +1,93 @@
+import moment from 'moment'
+import { EthereumDate, EthereumEntityName, EthereumAsset } from '../models/generalModels'
+import { isNullOrEmpty } from '../../../helpers'
+import { toWei } from './transactionHelpers'
+import {
+  toEthUnit,
+  isValidEthereumAddress,
+  isValidEthereumPublicKey,
+  isValidEthereumTxData,
+  isValidEthereumPrivateKey,
+} from './cryptoModelHelpers'
+
+export function isValidEthereumDateString(str: string): str is EthereumDate {
+  if (isNullOrEmpty(str)) return false
+  return str.match(/^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{2}\d+$/i) !== null
+}
+
+export function isValidEthereumAsset(symbol: string): boolean {
+  if (
```

Comment:
I guess you can create an array of all symbols and then code it like if(ethereumAssetSymbols.include(symbol)

or use EThUnit from cryptoModels

Reply:
ah yes. The units should not be duplicated here - they are in EthUnit. This function should still accept a string but then check that the value against valid options in EthUnit
Also ethUnit should be an enum not a type. Then you can copy the isInEnum function from oreid-service into chainjs helpers and use it to check if a value is in the enum